doc/tutorials/core/basic_geometric_drawing/basic_geometric_drawing.rst

```diff
@@ -167,7 +167,7 @@ Explanation
         * The ellipse center is located in the point **(w/2.0, w/2.0)** and is enclosed in a box of size **(w/4.0, w/16.0)**
         * The ellipse is rotated **angle** degrees
         * The ellipse extends an arc between **0** and **360** degrees
-        * The color of the figure will be **Scalar( 255, 255, 0)** which means blue in RGB value.
+        * The color of the figure will be **Scalar( 255, 0, 0)** which means blue in RGB value.
```

Comment:
I say you should use BGR since basically you are telling that a figure will be drawn using the Scalar values, which represent the BGR color space in OpenCV and NOT the RGB color space.